addons/web/static/tests/notifications/notifications_tests.js

```diff
@@ -69,6 +69,51 @@ QUnit.test("can display a danger notification with a title", async (assert) => {
   assert.hasClass(notif.querySelector(".o_notification_icon"), "fa-exclamation");
 });
 
+QUnit.test("can display a notification with a button", async (assert) => {
+  const env = await makeTestEnv({ serviceRegistry });
+  const notifService = env.services.notification;
+  await mount(NotificationContainer, { env, target });
+
+  notifService.create("I'm a notification with button", {
+    buttons: [{
+      name: env._t("I'm a button"),
```

Comment:
I wouldn't call env._t here (this is not the purpose of this test, and you kinda rely on the fact that there is no translations in the test page to ensure that the test passes)